internal/encryption/integration_test.go

```diff
@@ -86,13 +86,26 @@ func TestEncryptionForFSM(t *testing.T) {
 	in.RunWithEncryption(t, "",
 		in.CopyModule("encryption"),
 		in.Deploy("encryption"),
+		// "Rosebud" goes from created -> paid to test normal encryption
 		in.Call[map[string]interface{}, any]("encryption", "beginFsm", map[string]interface{}{"name": "Rosebud"}, nil),
-		in.Sleep(3*time.Second),
-		in.Call[map[string]interface{}, any]("encryption", "transitionFsm", map[string]interface{}{"name": "Rosebud"}, nil),
-		in.Sleep(3*time.Second),
-
+		in.Sleep(2*time.Second),
+		in.Call[map[string]interface{}, any]("encryption", "transitionToPaid", map[string]interface{}{"name": "Rosebud"}, nil),
+		in.Sleep(2*time.Second),
 		validateAsyncCall("created", "Rosebud"),
 		validateAsyncCall("paid", "Rosebud"),
+
```

Comment:
This tests is for encryption, can we move these changes to a new test?

Reply:
The changes to this test are to make sure we are encrypting the payload in the fsm next event table